apps/passport-server/src/util/telegramHelpers.ts

```diff
@@ -155,3 +288,30 @@ export const dynamicEvents = async (
     }
   }
 };
+
+export const chatsToJoin = async (
+  ctx: BotContext,
+  range: MenuRange<BotContext>
+): Promise<void> => {
+  const db = ctx.session.dbPool;
+  if (!db) {
+    range.text(`Database not connected. Try again...`);
+    return;
+  }
+  const userId = ctx.from?.id;
+  if (!userId) {
+    range.text(`User not found. Try again...`);
+    return;
+  }
+
+  const events = await fetchEventsPerChat(db);
+  const eventsWithChats = await getEventsWithChats(db, ctx, events);
+  if (eventsWithChats && eventsWithChats.length === 0) {
+    range.text(`No groups to join at this time`);
+    return;
+  }
+  for (const chat of eventsWithChats) {
+    const proofUrl = generateProofUrl(userId.toString(), chat.ticketEventIds);
```

Comment:
After choosing their TG group, a user gets a prove link populated with that group's associated eventIds